graphql-server/go-server/graph/schema/datasource.graphqls

```diff
@@ -94,18 +69,17 @@ input ListDatasourceInput {
     fieldSelector: String
     page: Int
     pageSize: Int
-    # 搜索，name，displayname, description字段
     keyword: String
 }
 
 # mutation
-type Mutation {
+extend type Mutation {
     createDatasource(input: CreateDatasourceInput!): Datasource!
     updateDatasource(input: UpdateDatasourceInput): Datasource!
     deleteDatasource(input: DeleteDatasourceInput): Void
 }
 # query
-type Query{
+extend type Query{
     datasource(name: String!, namespace: String!): Datasource!
-    datasourcesPaged(input: ListDatasourceInput!): PaginatedDatasource!
+    datasourcesPaged(input: ListDatasourceInput!): PaginatedResult!
```

Comment:
similar to the Model below
```
getModel(name: String!, namespace: String!): Model!
listModels(input: ListModelInput!): PaginatedResult!
```
change it `datasource -> getDatasource`, `datasourcesPaged -> listDatasources`